core/arch/arm/plat-corstone1000/conf.mk

```diff
@@ -23,9 +23,18 @@ $(call force,CFG_PL011,y)
 $(call force,CFG_SECURE_TIME_SOURCE_CNTPCT,y)
 $(call force,CFG_ARM64_core,y)
 
-arm64-platform-cpuarch := cortex-a35
+# Default CPU core for Corstone1000 platform; override for other cores (e.g. cortex-a320)
+arm64-platform-cpuarch ?= cortex-a35
+
+ifeq ($(arm64-platform-cpuarch),cortex-a320)
+arm64-platform-cflags += -march=armv9.2-a
+arm64-platform-aflags += -march=armv9.2-a
+$(call force,CFG_ARM_GICV3,y)
+$(call force,CFG_CORSTONE1000_CORTEX_A320,y)
```

Comment:
Is this the best name for this CFG? I mean, if this configuration can be found on something else than Corstone1000 with Cortex-A320 then perhaps it should be called differently.